src/aspire/classification/class2d.py

```diff
@@ -41,3 +42,17 @@ def __init__(
         self.n_nbor = n_nbor
         self.n_classes = n_classes
         self.seed = seed
+
+    @abstractmethod
+    def classify(self):
+        """
+        Classify the images from Source into classes with similar viewing angles.
+
+        Returns classes and associated metadata (classes, reflections, distances)
+        """
+
+    @abstractmethod
+    def averages(self, classes, refl, distances):
+        """
+        Returns class averages using prescribed `aligner`.
```

Comment:
No aligner (or averager) defined at this stage, so maybe just “Returns class averages.”?

Reply:
Right, vestigial. Changed.